database_refresh/01-configmap-refresh-script.yaml

```diff
@@ -0,0 +1,21 @@
+---
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: db-refresh-script
+    data:
+      entrypoint.sh: |-
+        #!/bin/bash
+        set -e
+        # Refreshes only happen every other week, crontab doesnt really support this
```

Comment:
Just a thought. Given the CNL database is tiny, there is no harm in running this weekly (instead of bi-weekly) which means we can get rid of this week check, and thus allow us to run adhoc jobs anytime.

Reply:
Think it's better to be consistent with the other services - easier to reason about if everything gets rolled back together... there's something built into the chart to allow this to be override but not sure it's worth it as hopefully this won't last too long